web/template/login.gohtml

```diff
@@ -2,7 +2,7 @@
 <html lang="en" class="dark">
 <head>
     <title>TUM Live | Login</title>
-    {{template "headImports"}}
+    {{template "headImports" .VersionTag}}
```

Comment:
.VersionTag doesn't exist here, you have to get it using `NewIndexData` in `web/user.go` -> `func LoginPage(c *gin.Context)` otherwise the page does not work

Reply:
But we would technically need the VersionTag here as well to prevent caching issues, right? If we don't need it, I can simply remove the version tag.

No problem, I think this is something that should be tested thoroughly throughout the week, I will do one last merge then. :grin:
Also, I was kinda puzzled how much the `package.lock` file grew by adding `moment.js`. :thinking: Do you get the same `package.lock` size when installing the node dependencies on your system?

Reply:
The login page currently doesn't use any js, so it's probably not required there. But it could save us some headaches in the future to have it

Reply:
> Also, I was kinda puzzled how much the package.lock file grew by adding moment.js.

I don't think the issue is moment.js: 
```bash
$ npm init
$ npm i moment --save
$ npm i
$ wc -l *.json
> 14 package.json
> 31 package-lock.json
```

I suspect, webpacks dependency tree to be huge af

Reply:
> I suspect, webpacks dependency tree to be huge af

Yeah, probably. Still insane how much stuff (bloat? :D) is there.
> The login page currently doesn't use any js, so it's probably not required there. But it could save us some headaches in the future to have it

Ok, I'll add it then. :+1:  Also, is this normal that the `versionTag` on the test server is still `development`? Maybe I still messed up the deployment...

Reply:
> Yeah, probably. Still insane how much stuff (bloat? :D) there is. 

As long as they're dev dependencies it should be fine, that just seems to be what we have to live with using npm and webpack.

> Also, is this normal that the versionTag on the test server is still development

with #257 merged the versionTag will be the same as the container sha-tag (https://github.com/joschahenningsen/TUM-Live/pkgs/container/tum-live%2Ftest-server)